android/src/test/java/com/magicleap/magicscript/scene/nodes/dropdown/UiDropdownListNodeTest.kt

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (c) 2019 Magic Leap, Inc. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.magicleap.magicscript.scene.nodes.dropdown
+
+import android.content.Context
+import android.graphics.Typeface
+import androidx.test.core.app.ApplicationProvider
+import com.facebook.react.bridge.JavaOnlyMap
+import com.magicleap.magicscript.*
+import com.magicleap.magicscript.font.FontParams
+import com.magicleap.magicscript.font.FontProvider
+import com.magicleap.magicscript.icons.IconsRepository
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.whenever
+import org.amshove.kluent.shouldBe
+import org.amshove.kluent.shouldContain
+import org.amshove.kluent.shouldEqual
+import org.amshove.kluent.shouldNotBe
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.ArgumentMatchers.anyBoolean
+import org.mockito.ArgumentMatchers.anyString
+import org.robolectric.RobolectricTestRunner
+
+/**
+ * To represent node's properties map in tests we use [JavaOnlyMap] which
+ * does not require native React's resources.
+ */
+@RunWith(RobolectricTestRunner::class)
+class UiDropdownListNodeTest {
+
+    private lateinit var tested: UiDropdownListNode
+
+    private lateinit var item1: UiDropdownListItemNode
+    private lateinit var item2: UiDropdownListItemNode
+
+    private lateinit var context: Context
+    private lateinit var providerTypeface: Typeface
```

Comment:
no need to keep it here

Reply:
The variable is used in this test - `shouldUseTypefaceFromProvider`